haversine-processing-engine/src/utils/haversine.ts

```diff
@@ -0,0 +1,35 @@
+
+// TODO: Move to ES6+
+const deg2rad = function(num: number) {
+  return num * Math.PI / 180;
+}
+
+const distance = function(location1: number[], location2: number[], useMiles = false) { // lon lat
```

Comment:
In my OOP we'd create a class with a lon and lat property on it, that could be passed in here. but guess that's not da way? 

We'd usually like to put a return type here - but since this method isn't being export probably doesn't matter.